### PR TITLE
Feat/bot father

### PR DESCRIPTION
**Pull request by cardosofede**
PR to manage multiple bots from the dashboard 🚀

### CODE REVIEW

**Comment by fengtality** [comment]
When I created a bot, I ran into an error b/c it couldn't find a `conf_client.yml` file. I looked at the code in `docker_manager.py` and it appears that it should be copying a `conf_client.yml` file from the `/data_downloader/conf` folder and changing instance name to the name provided by the user. Therefore, I added a commit to add this `conf_client.yml` file.

Afterwards, I was able to use the broker to create an instance. However, when I attached to it, it didn't print a `MQTT Connected` message as in the video (https://www.youtube.com/watch?v=NKnWRpq9qW8) and I saw this in the Dashboard.
Screen Shot 2023-06-14 at 3 45 25 PM

I suspect my broker isn't connected to the bot

**Comment by fengtality** [comment]
After the last commit, bot instances should now auto-connect to Broker, allowing you to test the strategy start/stop controls.

To test a script, copy [simple_pmm_example.py](https://github.com/hummingbot/hummingbot/blob/master/scripts/simple_pmm_example.py) into the `hummingbot_files/bot-configs/[instance-name]/conf/strategies` folder. 

You may need to stop and start the instance, the bot can only access script available at startup. 

Afterwards, you should be able to start, get status, and stop the script from the dashboard!

**Comment by nikspz** [comment]
Latest commit [1658290](https://github.com/hummingbot/dashboard/pull/31/commits/1658290cf385df2cff5394e467309a5d5561affe): 

Test performed:
- Created bot1,bot2,bot3 successfully using dashboard
- Review user could stop and exit bots successfully
- started dca_example.py  using dashboard successfully
- created bot1, bot2, bot3 successfully
- started dca_example.py  using dashboard successfully
- check status successfully
- attached bot1,bot2 review script actually running


redundant .yml file in strategy start:
Steps:
1. copy papertrade.yml strategy to hummingbot_files\bot_configs\hummingbot-bot3\conf\strategies folder
2. exit and start the bot3
3. Run strategy from dashboard

failed to start the .yml file from dashboard

image
In the client:
image

File: hbotrc/__init__.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+__version__ = "0.1.0"
```
Comment (devamin) [inline hbotrc/__init__.py:1]: I would prefer to install it as a package in environment_conda.yml
```
name: dashboard
channels:
  - conda-forge
dependencies:
  - python=3.9
  - sqlalchemy
  - pip
  - pip:
      - ccxt
      - streamlit
      - watchdog
      - plotly
      - pycoingecko
      - glom
      - defillama
      - statsmodels
      - pandas_ta
      - pyyaml
      - commlib-py
      - git+https://github.com/hummingbot/hbot-remote-client-py.git

```

Comment (cardosofede) [reply hbotrc/__init__.py:1]: yes I agree, but in this initial draft I made some quick changes to the code and I was testing how to run async tasks in streamlit but definitely we need to use the same repo.

Comment (cardosofede) [reply hbotrc/__init__.py:1]: perfect! Tomorrow we have the first live session. I think that @fengtality is going to create some tasks around how we can start working together on this.
from my side I will clean up this branch and probably we can merge it so people can start using it and then we can split our work in different pages

**Review by fengtality — APPROVED** [review]

**Pull request merged**